docs/admin_api/rooms.md

```diff
@@ -59,6 +59,7 @@ The following fields are possible in the JSON response body:
     - `guest_access` - Whether guests can join the room. One of: ["can_join", "forbidden"].
     - `history_visibility` - Who can see the room history. One of: ["invited", "joined", "shared", "world_readable"].
     - `state_events` - Total number of state_events of a room. Complexity of the room.
+    - `room_type` - the type of the room. It is `null` for normal rooms and `m.space` for spaces.
```

Comment:
Note that there exist room types other than `m.space` as well. Just searching matrix.org's database I find:

* `io.element.video`
* `org.matrix.msc3815.profile`
* `org.matrix.msc3815.world`
* `msc3588.stories.stories-room`

While the Matrix spec only defines a room type of `m.space` currently, unstable identifiers as room types are permitted, and more stable types will inevitably make their way into the way in the future: https://spec.matrix.org/v1.3/client-server-api/#types

So I'd avoid saying the value may *only* be `m.space` or null. Rather, `m.space` should simply serve as one such example of a value.